lib/Pinto/Action/Look.pm

```diff
@@ -0,0 +1,75 @@
+# ABSTRACT: Force a package to stay in a stack
+
+package Pinto::Action::Look;
+
+use Moose;
+use MooseX::StrictConstructor;
+use MooseX::MarkAsMethods ( autoclean => 1 );
+use MooseX::Types::Moose qw(Str);
+
+use Pinto::Util qw(throw);
+use Pinto::Types qw(TargetList);
+use Pinto::Shell;
+
+use Path::Class qw(file);
+
+#------------------------------------------------------------------------------
+
+# VERSION
+
+#------------------------------------------------------------------------------
+
+extends qw( Pinto::Action );
+
+#------------------------------------------------------------------------------
+
+has targets => (
+    isa      => TargetList,
+    traits   => [qw(Array)],
+    handles  => { targets => 'elements' },
+    required => 1,
+    coerce   => 1,
+);
+
+has shell => (
+    is       => 'ro',
+    isa      => Str,
```

Comment:
Should probably use the `File` type from `Pinto::Types` here. Be sure to set `coerce => 1`